content/post/Matter_future.md

```diff
@@ -0,0 +1,127 @@
++++
+title = 'Matter and the future'
+slug = 'Matter_future'
+image = '/static/images/Cover.jpeg'
+date = "2023-03-23T10:00:00"
+author = 'Tushar Bhatia'
+tags=['IoT', 'Networking', 'Smart Home', 'Smart Devices', 'Google-projects']
+categories =['IoT', 'Smart-Devices', 'Smart Homes', 'Networks']
+description = 'This blog explores Matter a new communication protocol made by CSA'
+disableComments = false
++++
+
+Hi everyone, This blog explores the new communication protocol, known as Matter which is being developed by <strong>Connectivity Standards Alliance (CSA)</strong>
+<br>
+To know more, Let's go on a journey and explore the new generation of communication protocols.
+<br>
+
+  ## What's Matter?
+  As the IoT market boomed. The popularity of smart home devices shot up. This resulted in a new market for Smart Home. Many companies joined hands and contributed to this boom by making Smart Devices.
+  <br>
+  <br>
+  As there were a large number of devices, our general communication protocols such as Wifi and Bluetooth couldn't keep up. To provide a solution for this new protocols like Z-wave, emerged. But over time, they were also hit by the demand.
+  This resulted in a huge wave of devices each having a different communication protocol, and requiring a hub to function.
+  With time, a new revolution was long overdue. Google joined hands with Amazon, Apple, and Zigbee Alliance together forming a new group known as CSA. To revolutionize the communication protocol.
+  <br>
+  <br>
+  They sought to develop a unified solution. All the major companies that we know today, joined hands in this journey to build a unified solution for IoT devices.
+  This was known as <b>Project CHIP( Connected Home Over IP)</b>. It evolved into what we know today as <b>Matter</b>.
+  </p>
+  
+  ## Why Matter should matter?
```

Comment:
When you remove the unnecessary tags, please also de-indent everything in the file.